glean-core/src/debug.rs

```diff
@@ -0,0 +1,216 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! # Debug options
+//!
+//! The debug options for Glean may be set by calling one of the `set_*` functions
+//! or by setting specific environment variables.
+//!
+//! The environment variables will be read only once when the options are initialized.
+//!
+//! The possible debugging features available out of the box are:
+//!
+//! * **Ping logging** - logging the contents of ping requests that are correctly assembled;
+//!         This may be set by calling glean.set_log_pings(value: bool)
```

Comment:
This function will come in the next PR.

Reply:
Do you mean add it to the code as a comment or add it here? I haven't opened a new bug for it, I was just going to do it as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1646706

Reply:
```suggestion
//!         This may be set by calling glean.set_log_pings(value: bool) (will be added by bug 1646706).
```

Reply:
I will open a PR adding this function immediatelly after I merge this one so I think this addition is not necessary.